fedimg/consumers.py

```diff
@@ -17,72 +18,117 @@
 # 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA
 #
 # Authors:  David Gay <dgay@redhat.com>
-#
+#           Sayan Chowdhury <sayanchowdhury@fedoraproject.org>
+"""
+This is the `fedmsg consumer`_ that subscribes to the topic emitted after the
+completion of the nightly and production compose. The consumer on receving the
+message uploads the image using the API of the cloud providers.
+"""
 
 import logging
-log = logging.getLogger("fedmsg")
-
 import multiprocessing.pool
 
 import fedmsg.consumers
 import fedmsg.encoding
 import fedfind.release
 
 import fedimg.uploader
-from fedimg.util import get_rawxz_urls, safeget
 
+from fedimg.config import PROCESS_COUNT, STATUS_FILTER
+from fedimg.utils import get_rawxz_urls, get_value_from_dict
 
-class FedimgConsumer(fedmsg.consumers.FedmsgConsumer):
-    """ Listens for image Koji task completion and sends image files
-        produced by the child createImage tasks to the uploader. """
+LOG = logging.getLogger("fedmsg")
```

Comment:
I was told multiple times that it's better to use ``__name__`` in the logger here:)

Reply:
Yes, I was told too during Flock this year. So will change to __name__